lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -9,22 +9,57 @@
 This means that, if your charm is related to, for example, COS' Tempo charm, you will be able to inspect
 in real time from the Grafana dashboard the execution flow of your charm.
 
-To start using this library, you need to do two things:
+# Quickstart
+Fetch the following charm libs (and ensure the minimum version/revision numbers are satisfied):
+
+    charmcraft fetch-lib charms.tempo_k8s.v2.tracing  # >= 1.10
+    charmcraft fetch-lib charms.tempo_k8s.v1.charm_tracing  # >= 2.7
+
+Then edit your charm code to include:
+
+```python
+# import the necessary charm libs
+from charms.tempo_k8s.v2.tracing import TracingEndpointRequirer, charm_tracing_config
+from charms.tempo_k8s.v1.charm_tracing import charm_tracing
+
+# decorate your charm class with charm_tracing:
+@charm_tracing(
+    # forward-declare the instance attributes that the instrumentor will look up to obtain the
+    # tempo endpoint and server certificate
+    tracing_endpoint="charm_tracing_endpoint",
```

Comment:
Should it be tracing_endpoint (without charm_) as the properties are named below in __init__? Same with server_cert.